examples/ping-pong/src/main.rs

```diff
@@ -193,6 +193,8 @@ async fn main_loop(agent: &GraphcastAgent, running: Arc<AtomicBool>) {
                 .await
             {
                 error!(error = tracing::field::debug(&e), "Failed to send message");
+            } else {
+                info!("Ping message sent successfully")
```

Comment:
perhaps lower the level for this to debug? either way is okay I guess, since this is just ping-pong